src/tests/Microsoft.PowerFx.Interpreter.Tests/SerializationTests.cs

```diff
@@ -95,16 +97,32 @@ public void OptionSetDefaultExpressionValueErrorTests()
             var result = check.GetEvaluator().Eval();
 
             Assert.IsType<ErrorValue>(result);
-        }
+        }       
 
-        private class BooleanOptionSet : OptionSet, IExternalOptionSet
-        {
+        internal class BooleanOptionSet : OptionSet, IExternalOptionSet
```

Comment:
>BooleanOptionSet [](http://example.com/codeflow?start=23&length=16)

Not for this PR but this class seems useful not only for tests. Maybe we could move it to a helper class or something. Just an idea.

Reply:
Agreed.